app/models/user.rb

```diff
@@ -356,19 +356,11 @@ def destroy
   end
 
   def military_information
-    @military_information ||= if Flipper.enabled?(:military_information_vaprofile)
-                                FormProfile.new(form_id: nil, user: self).initialize_military_information
-                              else
-                                EMISRedis::MilitaryInformation.for_user(self)
-                              end
+    @military_information ||= FormProfile.new(form_id: nil, user: self).initialize_military_information
```

Comment:
I think you can remove the entire `military_information` function from the User model actually. I searched the codebase and don't see any usages of it

Reply:
Thanks for spotting this, Trevor. The method has been fully removed, along with corresponding specs.